### DISCUSSION TITLE
How to completely remove on new M1 MacBooks?

### DISCUSSION THREAD

**Discussion by joranv1** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#144] [Q&A]
I read I have to set it back to 100% before removing it and that I can reset the SMC to be sure but the new M1 Macs don't have an SMC. How do I know if it is completely uninstalled without any "leftovers"? If I set it to 100% before removing it, does that disable the (always enabled) optimized charging on the new M1 MacBooks that prevents them from charging to 100% (so they can only charge to 98-99%)?

**Comment by MatthiasKerbl** [comment]
Hi @MrBananaPants,
to uninstall AlDente on Apple Silicon Macbooks follow these steps:
1. Set Max. Battery Charge to 100% and wait 1-2 minutes for changes to take place.
2. Quit the app and delete the program file
3. Turn off your Macbook and close the lid for 30 seconds (this should reset your SMC)

**Reply by joranv1** [reply]
Hi, I followed your instructions but now the battery charges to 100% instead of max at 98-99% like it should be (Apple implemented this feature to decrease battery degradation). Would a reinstall of macOS fix this? I used CoconutBattery 2.

**Reply by MatthiasKerbl** [reply]
I think Apple does this in the background. It will show you 100% but it is actually at 98-99%.

**Reply by joranv1** [reply]
No, apps like CoconutBattery or better battery stats (or something like that) did always show 98-99%, never 100% (like in the status bar).

**Reply by MatthiasKerbl** [reply]
Interesting, do you have "Optimised Battery Charging" and "Manage Battery Longevity" activated in settings?

**Reply by joranv1** [reply]
No the M1 Macs don't have those options. They're enabled by default.

**Reply by MatthiasKerbl** [reply]
I see, I don't really know what else could be the reason for your issue. If you have done every step from above, everything from AlDente should be completely removed and so far, no one else has this issue.

**Reply by shooter366** [reply]
On my Intel MBP each macOS Update will reset all aldente changes. Therefore I guess it's enough to wait for the next update or to try the public beta.
